Wflow/test/run_sbm.jl

```diff
@@ -202,7 +202,7 @@ Wflow.run_timestep!(model)
     land = model.land
     @test land.atmospheric_forcing.precipitation[2] / precip[2] ≈ 2.0
     @test (land.atmospheric_forcing.potential_evaporation[100] - 1.50) / evap[100] ≈ 3.0
-    @test land.vegetation_parameters.leaf_area_index[100] / lai[100] ≈ 1.6
+    @test land.vegetation_parameters.leaf_area_index[100] / lai[100] ≈ 1.5999999403953562
```

Comment:
Would something like this be better? Because we do expect the value to be 1.6, but I guess it is slightly different due to floating point

```suggestion
    @test land.vegetation_parameters.leaf_area_index[100] / lai[100] ≈ 1.6 atol=1e-7
```